file.c

```diff
@@ -1621,11 +1621,15 @@ rb_file_directory_p(VALUE obj, VALUE fname)
 
 /*
  * call-seq:
- *   File.pipe?(file_name)   ->  true or false
```

Comment:
For all of these methods, they are primarily called with filenames, so the argument in the call-seq should remain `file_name`.